Client/core/CConnectManager.cpp

```diff
@@ -27,7 +27,8 @@ CConnectManager::CConnectManager()
     m_bIsConnecting = false;
     m_bSave = true;
     m_tConnectStarted = 0;
-
+    bool m_quickConnect{false};
```

Comment:
I don’t think you’re compiling your changes, because this shouldn’t be here, It should be in the header, and this file should remain unchanged.